doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -758,42 +801,205 @@ Response:
+(described in a section above).
+
+Response:
+
+* Successful operation uses 204 No Content
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
 
 + Parameters
     + entityId: Bcn_Welt (required, string) - Entity ID
-    + attrId: mistake (required, string) - Attribute to be retrieved.
+    + entityType: Room (required, string) - Entity Type
+    + attrName: temperature (required, string) - Attribute ID
 
-+ Response 404 (application/json)
++ Request (application/json)
 
         {
-            "error": "NotFound",
-            "description": "The entity does not have such an attribute"
+            "value": 25.0,
+            "unitCode": "CEL"
         }
 
-### Update an attribute value - JSON [PUT /v2/entities/{entityId}/attrs/{attrId}]
++ Response 200
+
+
+### Remove a single attribute [DELETE /v2/entities/type/{entityType}/id/{entityId}/attrs/{attrName}]
+
+Removes the attribute from the entity.
+
+Response:
+
+* Successful operation uses 204 No Content
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + entityType: Room (required, string) - Entity Type
+    + attrName: temperature (required, string) - Attribute ID
+
++ Response 204
+
+## Entity Attribute Value [/v2/entities/{entityId}/attrs/{attrName}/value]
+
+### Get attribute data [GET /v2/entities/{entityId}/attrs/{attrName}/value{?text}]
+
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">options</td>
+  <td>Operation opations. Allowed values: <code>text</code>.</td>
+</tr>
+</table>
+
+If the `text` option is not used, returns a JSON object which the `value` of the attribute. If the
+`text` option is used returns the attribute value in plain text.
+
+Response:
+
+* Successful operation uses 20O OK.
+* Error use a non-200 and (optionally) an error payload. See subsection on Error Responses for more detail.
 
 + Parameters
     + entityId: Bcn_Welt (required, string) - Entity ID
-    + attrId: temperature (required, string) - Attribute ID
+    + attrName: temperature (required, string) - Attribute to be retrieved.
+
++ Response 200 (application/json)
+
+        {
+            "value": 21.7
+        }
+
+### Update attribute value [PUT /v2/entities/{entityId}/attrs/{attrName}/value]
+
+The request payload is the value to update in the attribute, either a JSON object with a `value` property (in
+the case of using application/json as content type) or just the value (in the case of using text/plain as conent type).
+Other attribute data (type and metadata) are not touched.
+
```

Comment:
The request payload is the new attribute value. It can be represented as a JSON object (application/json), with a 'value' property, or as plain text (text/plain).